frontend/conf/riff-raff.yaml

```diff
@@ -8,9 +8,6 @@ deployments:
       amiTags:
         Recipe: xenial-membership
         AmigoStage: PROD
-      cloudFormationStackName: "Memb"
-      prependStackToCloudFormationStackName : false
-      appendStageToCloudFormationStackName : true
```

Comment:
As @davidfurey pointed out, adding this tagging in the AWS console is enough for Riff Raff to find the `Memb-PROD` stack:

![image](https://cloud.githubusercontent.com/assets/52038/25944394/0d5a1c4a-363b-11e7-95c5-3082b93f03a1.png)